apps/dcos_net/src/dcos_net_node.erl

```diff
@@ -0,0 +1,258 @@
+-module(dcos_net_node).
+-behaviour(gen_server).
+
+%% API
+-export([
+    start_link/0,
+    dump/0,
+    force_dump/0
```

Comment:
nit: Similarly instead of `force_dump/0` to something say `force_refresh/0`? I know there is another `force_refresh/0`, however, that is an internal function which we can name something else as compare to this one which is externally visible

Reply:
I've added `get_metadata/1` function with `force_refresh` option.